gulpfile.babel.ts

```diff
@@ -341,84 +318,106 @@ gulp.task('types', async () => {
   await exec('yarn typedefs');
 });
 
+let config: WebpackTaskOptions;
+const remoteCompatConfig = {
+  afterBuild: setupExamples,
+} as const;
+
 if (args.remote) {
-  gulp.task('sdk', () => {
-    return webpackTask({
-      devtool: 'remote',
-      disableMinification: true,
-      entry: './src/inboxsdk-js/inboxsdk-REMOTE',
-      destName: sdkFilename,
-      standalone: 'InboxSDK',
-    });
-  });
-  gulp.task(
-    'remote',
-    gulp.series('pageWorld', function impBundle() {
-      return webpackTask({
-        devtool: 'remote',
-        entry: './src/platform-implementation-js/main-INTEGRATED-PAGEWORLD',
-        destName: 'platform-implementation.js',
-        afterBuild: setupExamples,
-      });
-    })
-  );
-  gulp.task('default', gulp.series('sdk', 'remote'));
+  config = {
+    ...remoteCompatConfig,
+    devtool: false,
+    entry: {
+      [sdkFilename]: {
+        library: {
+          export: 'default',
+          name: 'InboxSDK',
+          type: OutputLibraryType.UMD,
+        },
+        import: './src/inboxsdk-js/inboxsdk-REMOTE',
+      },
+      'platform-implementation':
+        './src/platform-implementation-js/main-INTEGRATED-PAGEWORLD',
+    },
+    plugins: [
+      new webpack.SourceMapDevToolPlugin({
+        fileContext: '../',
+        // Adds hash to the end of the sourcemap URL so that
+        // remote code source maps are not cached erroneously by Sentry.
+        filename: '[file].map?hash=[contenthash]',
```

Comment:
After speaking with @Macil, these comments may be being stripped out by our pre-OSS SDK version that we have currently deployed.